CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 10.1
+* Add sponsor logo dimensions to answers (by bumping the content-api-models dependency)
```

Comment:
Not sure you need the `(by bumping the content-api-models dependency)`, this is transparent to the client.